test/e2e/angular-app/tslint.json

```diff
@@ -18,7 +18,6 @@
     "forin": true,
     "import-blacklist": [
       true,
-      "rxjs",
```

Comment:
curious why this got removed but not the other?

Reply:
This file was auto-modified by `ng upgrade`